### PR TITLE
config: add husky hooks for lint and prettier

### PR DESCRIPTION
**Pull request by abhishekthalatoty**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,23 @@
     "react-helmet": "^6.1.0"
   },
   "devDependencies": {
-    "prettier": "2.0.5"
+    "eslint": "^7.4.0",
+    "eslint-config-prettier": "^6.11.0",
```
Comment (pinnamanivenkat) [inline package.json]: We wouldn't want random package-lock.json updates. So, please remove all the ^ and just have normal version number

File: package.json [hunk 2]
```diff
@@ -19,7 +19,23 @@
...
+    "eslint-config-react-app": "^5.2.1",
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.11",
+    "prettier": "^2.0.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```
Comment (pinnamanivenkat) [inline package.json:31]: looks good!

File: package.json [hunk 3]
```diff
@@ -19,7 +19,23 @@
...
+    }
+  },
+  "lint-staged": {
+    "./src/*.{js,jsx}": [
```
Comment (pinnamanivenkat) [inline package.json]: add json and others as well.

**Review by pinnamanivenkat — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (turinghut) [reply package.json:35]: Should I add linting for Typescript files too?

Comment (abhishekthalatoty) [reply package.json]: fixed.

File: package.json (hunk 1 above)
Comment (abhishekthalatoty) [reply package.json]: fixed.

**Pull request merged**